[Bug]: Zoom in Image Standard Preview after using 360° Viewer

mcop1 opened the pull request:
closes #11052

mcop1 committed:
[Bug]: Zoom in Image Standard Preview after using 360° Viewer

kingjia90 commented:
Looks fine and ready to merge, but why not just moving the area max size setter inside the `initPreviewImage` (so it will stay similar to `initPreviewVr`, avoiding the extra method `setPreviewPanelSize`), something like this:
```
  this.displayPanel.on('resize', function () {
      if(this.previewMode == 'vr') {
          this.initPreviewVr();
      } else {
          this.initPreviewImage();
      }
  }.bind(this));
}
```
while changing `width` to `this.displayPanel.getWidth()` (same for the height, which already works similarly [here](https://github.com/pimcore/pimcore/blob/35c5e83d99c8244979b09817dac198ce1726d260/bundles/AdminBundle/Resources/public/js/pimcore/asset/image.js#L456)) like this:
```
initPreviewImage: function () {
    var html = '<img src="' + this.data.imageInfo['previewUrl'] + '">';
    ....
    var area = this.displayPanel.getEl().down('img');
    if(area) {
        area.setStyle('max-width', (this.displayPanel.getWidth() - 340) + "px");
        area.setStyle('max-height', (this.displayPanel.getHeight() - 40) + "px");
    }
},
```
so we could clean these legacy(?) parameters on the resize event 
https://github.com/pimcore/pimcore/blob/35c5e83d99c8244979b09817dac198ce1726d260/bundles/AdminBundle/Resources/public/js/pimcore/asset/image.js#L434

wdyt?

mcop1 committed:
[Bug]: Zoom in Image Standard Preview after using 360° Viewer

mcop1 committed:
[Bug]: Zoom in Image Standard Preview after using 360° Viewer

mcop1 commented:
@kingjia90 you´re right, I changed it. Regarding the legacy parameters, I don´t know where they come from, but since they are not  used anywhere I removed them.
